src/common/snippets/src/lowered/pass/validate_shapes.cpp

```diff
@@ -7,6 +7,7 @@
 #include "snippets/lowered/linear_ir.hpp"
 #include "snippets/op/loop.hpp"
 #include "snippets/shape_inference/shape_inference.hpp"
+#include "snippets/utils.hpp"
```

Comment:
Minor: I believe it's not really needed, is it?

Reply:
Correct. It was needed for the recent code that has been removed. Thanks!